content/20.treatments.md

```diff
@@ -544,10 +544,17 @@ These last findings highlight the need to search for a balance between impairing
 This aspect should be investigated further in upcoming trials.
 
 Perhaps, the TCZ treatment would best suit patients with severely compromised lungs due to the COVID-19 infection and are therefore at greater risk of death, in order to stop the uncontrolled immune response before it’s too late.
-<!-##### Summary
 
-Summarize the state of the symptom management approach.
--->
+**Summary.**
+On April 29 of the current year UC San Diego Health started phase III clinical trial to establish whether TCZ, a drug commonly used to treat rheumatoid arthritis and similar diseases, also has therapeutic effect on COVID-19 patients (https://ucsdnews.ucsd.edu/pressrelease/arthritis-drug-presents-promise-as-treatment-for-covid-19-pneumonia).
+Approximately, 25% of coronavirus patients develop (ARDS) which is caused by an excessive early response of the immune system, also known as the cytokine storm [doi.org/10.1186/s12967-020-02339-3].
+Such overwhelming inflammation is triggered by the IL-6.
+In this context, TCZ is an inhibitor of IL-6 and can, therefore, neutralise the inflammatory pathway that takes to the cytokine storm.
+According to the health commission of China, TCZ can be used in patients with high concentration of IL-6 in their plasma ( 20 pg/ml) or with extensive lung damages.
+The recommended dose is 400mg to be diluted in 100 ml containing 0.9% NaCl.
+In case of poor effectiveness it is possible to administrate another dose after 12h from the first administration.
```

Comment:
Dear @rando2 ,

Here is the text of the paper in the Journal of translation medicine [https://doi.org/10.1186/s12967-020-02339-3] that I summarised in our review:

The immunotherapy strategy about Tocilizumab treatment has been formally included in the diagnosis and treatment program of COVID-19 (7th edition) of the national health commission of China since 3th March 2020 as following: Tocilizumab can be used in patients with extensive bilateral lung lesions opacity or in severe or critical patients, who have elevated laboratory detected IL-6 levels. The first dose is 4–8 mg/kg (the recommended dose is 400 mg, diluted to 100 ml with 0.9% normal saline, and the infusion time is more than 1 h). For patients with poor initial efficacy, an additional application can be made after 12 h (the dose is the same as before). The maximum number of times of administration is two, and the maximum dose of a single dose should not exceed 800 mg. Note that patients with allergic reactions, such as tuberculosis and other active infection are contraindicated. We suggest that IL-6 concentrations can be detected if fever persists for more than 3 days. By chemiluminescence detection, if serum IL-6 content is over 20 pg/ml, Tocilizumab can be used.

Yes, the dose it was determined by body weight 4-8mg/kg, but the recommended dose is 400mg. Then if patient's condition does not improve you can administrate another does after 12 h

Reply:
@aadattoli Thank you! Given that text, I think the way you have it makes sense (on line 554)